src/components/elements/upload/UploaderBase.tsx

```diff
@@ -231,7 +231,11 @@ const Uploader: React.FC<UploaderProps> = ({
             setCurrentUpload(res);
             onUpload(res, file);
           })
-          .then(() => setLoading(false));
+          .then(() => setLoading(false))
+          .catch((error) => {
+            setLoading(false);
+            setError(error.message);
+          });
       }
```

Comment:
What's the expected behavior here? When I try to upload an image, the `CreateDirectUploadMutation` succeeds but the "save" button remains disabled. This is probably just because I don't have s3 setup locally, but just wanted to check.

Reply:
For me, before this change, trying to upload an image causes a 500 error in the console (because of not having s3 locally setup) but the modal continues spinning on the "loading" animation after the error.

With this change, after it errors out the modal shows "Response not successful: Received status code 500" and the save button is still disabled.
<img width="638" alt="Screenshot 2024-04-05 at 4 34 14 PM" src="https://github.com/greenriver/hmis-frontend/assets/16002775/df9d238f-64eb-4c54-8054-a86df04cb1e8">